### PR TITLE
Backport of 81X PU jet ID

### PR DESCRIPTION
**Pull request by ahinzmann**
Backport of #14723 and #15119.

This changes MiniAOD content, so unless we start a new MiniAOD campaign at some point, this does not go into the 80X release.

However, it would be nice if cmsbuild could test this, since we may distribute this as analysis recipe.

### CODE REVIEW

**Comment by slava77** [comment]
@ahinzmann 
was the weight file PR made already ( request to have the weight files to be in 80X)?
Please confirm, it's needed before this PR can be tested.

**Comment by ahinzmann** [comment]
I'm not sure, since there was no answer in this thread:
https://github.com/cms-data/RecoJets-JetProducers/pull/9

**Comment by cvuosalo** [comment]
@cmsbuild please test

**Comment by ahinzmann** [comment]
Closing this PR for analysis recipe after tests passed. May only be re-opened in case of a new MiniAOD campaign in 80.

**Comment by ahinzmann** [comment]
If coordination decides on a full re-reco of data AND MC in 80X, this PR should be included in 80X.

**Comment by ahinzmann** [comment]
Does this still merge and pass tests?

**Comment by slava77** [comment]
@cmsbuild please test

... still merges

**Comment by cvuosalo** [comment]
backport #14723

File: RecoJets/JetProducers/python/PileupJetID_cfi.py [hunk 1]
```diff
@@ -10,8 +10,9 @@
 _chsalgos_74x = cms.VPSet(full_74x_chs,cutbased)
 _chsalgos_76x = cms.VPSet(full_76x_chs,cutbased)
 _chsalgos_80x = cms.VPSet(full_80x_chs,cutbased)
+_chsalgos_81x = cms.VPSet(full_81x_chs,cutbased)
 
-_stdalgos    = _chsalgos_80x
+_stdalgos    = _chsalgos_81x
```
Comment (cvuosalo) [inline RecoJets/JetProducers/python/PileupJetID_cfi.py]: Is it OK to set the _stdalgos to the 81X version in 80X?

**Comment by cvuosalo** [comment]
backport #15119

**Comment by cvuosalo** [comment]
+1

For #15211 fe554d718fc1802a10c5308c6a9438040f1538b6

Adding eta-specific MVA training files for PU jet ID and bug fixes to pile-up jet ID quantities. There should be no change in upstream RECO quantities.

Backport of 81X PRs #14723 and #15119, which are both merged.

The code changes match those in the original PRs, and Jenkins tests against baseline CMSSW_8_1_X_2016-07-14-1100 show only small changes in the PU jet ID MVA output, as would be expected. See #15119 for more details.

File: RecoJets/JetProducers/python/PileupJetID_cfi.py (hunk 1 above)
Comment (ahinzmann) [inline RecoJets/JetProducers/python/PileupJetID_cfi.py]: the "81X" training is based on 80X RECO. The naming "81X" may indeed confuse. It is rather "80XvarFix", like the name of the weight files. However, this name is never visible anywhere to the user.
Is it worth changing the name, but breaking consistency of 80X and 81X releases?

**Pull request merged**